perun-core/src/test/java/cz/metacentrum/perun/core/entry/GroupsManagerEntryIntegrationTest.java

```diff
@@ -5085,6 +5085,89 @@ public void deleteGroupsFromAutoRegistration() throws Exception {
 		assertEquals(Collections.emptyList(), groupsManager.getGroupsForAutoRegistration(sess, vo));
 	}
 
+	@Test
+	public void indirectMembershipPathSimple() throws Exception {
+		System.out.println(CLASS_NAME + "indirectMembershipPath");
```

Comment:
Should be `System.out.println(CLASS_NAME + "indirectMembershipPathSimple");`

Reply:
fixed